### PR TITLE
Changed AdoptionUI to ShelterUI

### PR DESCRIPTION
**Pull request by gracebarsny**
AdoptionUI.prefab -> ShelterUI.prefab
PPAdoptionUIController.cs -> PPShelterUIController.cs

### CODE REVIEW

File: Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    protected override void fetchReferences()
+    {
+        base.fetchReferences();
+        database = DogDatabase.GetInstance;
+        populateAvailableDogs(database);
+    }
+
+    #endregion
+
+    void populateAvailableDogs(DogDatabase database)
```
Comment (imann24) [inline Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs]: Confirming we want a new random selection each time the script runs? Not a random selection every realtime day or something more complex

Comment (gracebarsny) [inline Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs]: The current design is 4 new dogs every 24 hours.

Comment (imann24) [inline Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs]: @gracebarsny Can you override to use a random seed that is somehow generated by the realtime day: 
- Random Seed: https://msdn.microsoft.com/en-us/library/ctssatww(v=vs.110).aspx
- Realtime Day: https://msdn.microsoft.com/en-us/library/system.datetime.day(v=vs.110).aspx

I can see if I can add a quick PR w/ a subclass of RandomBuffer containing the above functionality

Comment (gracebarsny) [inline Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs]: Ok

**Comment by imann24** [comment]
This is super nitpicky, and also not likely your fault, because I think I originally made this UI, but the text is truncated on 9:16 aspect ratios: 
screen shot 2017-01-11 at 5 03 07 pm

**Pull request merged**